ui/pages/institutional/confirm-add-custodian-token/confirm-add-custodian-token.js

```diff
@@ -79,81 +79,41 @@ const ConfirmAddCustodianToken = () => {
     ).value;
   }
 
+  const custodian = findCustodianByDisplayName(custodianLabel);
+
   return (
     <Box className="page-container">
-      <Box className="page-container__header">
-        <Text className="page-container__title">{t('custodianAccount')}</Text>
-        <Text className="page-container__subtitle">
-          {t('mmiAddToken', [connectRequest.origin])}
+      <Box paddingTop={6} paddingLeft={4} paddingRight={4}>
+        <Text>
+          <Chip
+            borderColor={BorderColor.borderMuted}
+            label={connectRequest.origin}
+            maxContent={false}
+            leftIconUrl={custodian?.icon}
+            labelProps={{
+              textAlign: TextAlign.Center,
+            }}
+          />
         </Text>
       </Box>
       <Box padding={4} className="page-container__content">
-        {custodianLabel && (
-          <>
-            <Text padding={4} color={TextColor.textDefault}>
-              {t('custodian')}
-            </Text>
-            <Label
-              marginRight={4}
-              marginLeft={4}
-              color={TextColor.textAlternative}
-              className="add_custodian_token_confirm__url"
-            >
-              {custodianLabel}
-            </Label>
-          </>
-        )}
-
-        <Text padding={4} color={TextColor.textDefault}>
-          {t('token')}
+        <Text
+          padding={4}
+          fontWeight={FontWeight.Bold}
+          variant={TextVariant.headingSm}
+          as="h5"
```

Comment:
Optional: you can remove `as` props now, check https://consensys.slack.com/archives/GTQAGKY5V/p1690998425185369

Reply:
Thank you!